src/scripts/components/VideoOverlay.js

```diff
@@ -247,28 +244,28 @@ class VideoOverlay extends Component<Props> {
             showShareModal={showShareModal}
           >
             {isEmbed && (
-              <ProfileButtonWrapper>
-                <IconButton
-                  color={
-                    activePlugin === PLAYER_PLUGIN.WALLET ? Colors.purple : ''
-                  }
-                  data-test-id="wallet-info-button"
-                  icon="/assets/img/profile.svg"
-                  onClick={(e: Object) => {
-                    e.stopPropagation()
-                    toggleActivePlugin(PLAYER_PLUGIN.WALLET)
-                  }}
-                />
+              <ProfileButtonWrapper
+                warn={activePlugin !== PLAYER_PLUGIN.WALLET}
```

Comment:
Is `warn` the correct adjective here? Maybe we should continue renaming this; I don't think if the active plugin is not the wallet that we are trying to indicate a warning w/ this button, but perhaps I'm forgetting what it looks like